index.html

```diff
@@ -553,6 +561,10 @@
           <a href="https://www.w3.org/TR/ldp/#ldprs-put-servermanagedprops">4.2.4.3</a>, 4xx responses MUST be
           accompanied by a <code>Link</code> header with <code>http://www.w3.org/ns/ldp#constrainedBy</code> relation.
         </p>
+        <p id='external-content-want-digest'>
+          <code>GET</code> and <code>HEAD</code> requests to any external binary content <a>LDP-NR</a> MUST correctly
+          respond to the <code>Want-Digest</code> header defined in [[!RFC3230]].
```

Comment:
I like requiring this, and think it will make it more usable for clients.  However, if common webservers (e.g., Apache and Nginx) don't support Want-Digest/Digest, an implementation would need to fetch the content and checksum it in order to generate these headers.  So that would mean fetching the entire content twice to serve it the first time.  Is this acceptable overhead?

Reply:
I think this is acceptable overhead now that the requirement for support has been changed to SHOULD

Reply:
@escowles as implementation work goes forward, we should be trying to identify patterns here. Some of the use cases that motivate external storage are also going to make pre-emptive checksumming especially punitive.